src/http/server.cr

```diff
@@ -169,10 +289,12 @@ class HTTP::Server
   def close
     @wants_close = true
     @processor.close
-    if server = @server
-      server.close
-      @server = nil
+
+    @interfaces.each do |interface|
+      interface.close
```

Comment:
Closing an interface can raise, which will prevent further interfaces from being closed.

Reply:
Will wrap it in rescue. We can probably ignore any exceptions, right?